.github/workflows/unit-tests.yml

```diff
@@ -130,7 +130,7 @@ jobs:
           [[ "${{ matrix.mode }}" = *-deps ]] && mv composer.json.phpunit composer.json || true
 
           if [[ "${{ matrix.mode }}" = low-deps ]]; then
-            echo SYMFONY_PHPUNIT_REQUIRE="nikic/php-parser:^4.18" >> $GITHUB_ENV
+            echo SYMFONY_PHPUNIT_REQUIRE="nikic/php-parser:^5.0" >> $GITHUB_ENV
```

Comment:
PHPUnit 11 (through its dependency on `phpunit/php-code-coverage` is not compatible with `nikic/php-parser` 4